Maths/AverageMedian.js

```diff
@@ -1,14 +1,5 @@
-/*
- * Median: https://en.wikipedia.org/wiki/Median
- *
- * function averageMedian
- * to find the median value of an array of numbers
- * the numbers in an array will be sorted in ascending order by the function sortNumbers
- * if the length of the array is even number, the median value will be the average of the two middle numbers
- * else if the length of the array is odd number, the median value will be the middle number in the array
- */
-
-const averageMedian = (numbers) => {
+const averageMedian = (_numbers) => {
+  let numbers = [..._numbers];
```

Comment:
I don't understand why any of the changes other than this line is required. Keep this line and revert all the other changes.